UnitsNet.WindowsRuntimeComponent/CustomCode/UnitAbbreviationsCache.cs

```diff
@@ -45,10 +46,16 @@ static UnitAbbreviationsCache()
 
         private void LoadGeneratedAbbreviations()
         {
-            foreach(var localization in GeneratedLocalizations)
+            var quantities = typeof(Length)
+                .Wrap()
+                .Assembly
+                .GetExportedTypes()
+                .Where(t => typeof(IQuantity).Wrap().IsAssignableFrom(t) && t != typeof(IQuantity));
```

Comment:
I'm thinking it might be better to just statically generate a `Type` array with the 100ish quantity types, since this will only pick up our own IQuantity types. Reflection is perhaps better suited for picking up types externally or not known at compile time. It is also not particularly performant, so I assume it will have some minor impact on startup time compared to a static list. 

What do you think?

Reply:
I see a couple of other places with the same reflection to look up our own quantity types:
https://github.com/angularsen/UnitsNet/search?q=GetExportedTypes

Maybe we can just have an internal static property to reuse the lookup of our quantity types.

Reply:
I think that's a good suggestion. Can I do as a separate PR that hits all places?

Reply:
Absolutely, seem slike you already got the fix in though! That was quick. Let's merge this, and address any remaining places later.

Reply:
I changed my mind, I did it now 😆 

Easy enough